src/components/ui/AssetsPanel.tsx

```
@@ -1,318 +1,686 @@
+import { useEffect, useState, useCallback, useMemo } from 'react'
+import { X, Image as ImageIcon, RotateCcw, Edit2, Box, Grid, Layers } from 'lucide-react'
+import { db, GeneratedAsset } from '@/lib/client-db'
+import type { Generated3DAsset } from '@/lib/types/3d-generation'
+import { Button } from '@/components/ui/button'
+import { ANIMATION_PRESET_LABELS, type AnimationPreset } from '@/lib/types/3d-generation'
+import { cn } from '@/lib/utils'
+
+/**
+ * Constants
+ */
+const SKYBOX_ASSET_SUFFIX = SKYBOX_ASSET_SUFFIX
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)